src/chainparams.h

```diff
@@ -115,9 +115,9 @@ class CChainParams
     bool fAllowMultiplePorts;
     CCheckpointData checkpointData;
     ChainTxData chainTxData;
-    int nPoolMinParticipants;
-    int nPoolMaxParticipants;
-    int nFulfilledRequestExpireTime;
+    int nPoolMinParticipants{0};
```

Comment:
As asked by @UdjinM6 in Slack, please remove the default initialization and move the initialization into individual chain params to keep consistency with other code.

Reply:
FYI, Only this part is mean...the default initialization in `BIP9Deployment` is fine IMHO.

Reply:
just removed the default initialization, isn't it already initialized in the same way the other variable are at chainparams.cpp L355 L532 and L687?